test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.test.cpp

```diff
@@ -1120,6 +1120,107 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit, CircularEquatorial)
     }
 }
 
+TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit, GeoSynchronous)
+{
+    using ostk::core::ctnr::Array;
+    using ostk::core::fs::Path;
+    using ostk::core::fs::File;
+
+    using ostk::math::obj::VectorXd;
+
+    using ostk::physics::units::Length;
+    using ostk::physics::units::Angle;
+    using ostk::physics::time::Scale;
+    using ostk::physics::time::Instant;
+    using ostk::physics::time::DateTime;
+    using ostk::physics::time::Time;
+    using ostk::physics::Environment;
+
+    using ostk::astro::trajectory::Orbit;
+    using ostk::astro::trajectory::State;
```

Comment:
Move to the top of the file if not already defined there. Guidelines on these is to define `using` statement at the top now fyi